src/Dapr.Client/DaprClient.cs

```diff
@@ -223,7 +226,7 @@ public async ValueTask<StateEntry<TValue>> GetStateEntryAsync<TValue>(string sto
         public abstract ValueTask<bool> TryDeleteStateAsync(
             string storeName,
             string key,
-            ETag etag,
+            string etag,
```

Comment:
I don't have a big objection to this, but the rationale for making ETag its own type is to make it clear to users that they **should not** try to guess a good value for an etag. Making it a simple string makes it appear like something users can create.

Reply:
I agree with the rationale, will add it back and make ti a struct.